packages/opentelemetry-core/src/metrics/NoopMeter.ts

```diff
@@ -85,9 +86,9 @@ export class NoopMetric<T> implements Metric<T> {
 
   /**
    * Removes the Handle from the metric, if it is present.
-   * @param labelValues the list of label values.
+   * @param labels the object of label set.
    */
-  removeHandle(labelValues: string[]): void {
+  removeHandle(labels: LabelSet): void {
     return;
```

Comment:
should there be a TODO here to implement this?

Reply:
+1 for TODO
There is also ongoing discussion on whether or not we are going to include `removeHandle` in the API.
https://github.com/open-telemetry/opentelemetry-specification/pull/299#issuecomment-542934491